sdk/compute/Azure.ResourceManager.Compute/src/Customize/Models/VirtualMachinePatch.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Compute.Models
+{
+    /// <summary> Describes a Virtual Machine Update. </summary>
```

Comment:
well let us remove this xml document to avoid that in the future we change the description of this model and two partial classes have different xml document and lead to funny results.